backend/pokemon/models.py

```diff
@@ -5,6 +5,6 @@
 class Pokemon(models.Model):
     pokemon_id = models.IntegerField(null=True)
     name = models.CharField(max_length=200, null=True)
-    attack = models.CharField(max_length=200, null=True)
-    defense = models.CharField(max_length=200, null=True)
-    hp = models.CharField(max_length=200, null=True)
+    attack = models.IntegerField(null=True)
+    defense = models.IntegerField(null=True)
+    hp = models.IntegerField(null=True)
```

Comment:
Sorry if you already discussed this with Rebeca, but does all fields here need to be `null=True`?

Reply:
Actually, they don't. I'll change this. Thanks!